Sources/RichTextKit/Attributes/RichTextLayoutManager.swift

```diff
@@ -0,0 +1,70 @@
+#if os(macOS)
+import AppKit
+#else
+import UIKit
+#endif
+
+public final class RichTextLayoutManager: NSLayoutManager {
+    public override init() {
+        super.init()
+    }
+    
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+    
+#if os(macOS)
+    public override func processEditing(
+        for textStorage: NSTextStorage, 
+        edited editMask: NSTextStorageEditActions, 
```

Comment:
Cynical note: This is the only difference between macOS and iOS function signature :D